verification/curator-service/api/src/index.ts

```diff
@@ -86,6 +86,7 @@ apiRouter.put('/sources/:id([a-z0-9]{24})', sourcesController.update);
 apiRouter.delete('/sources/:id([a-z0-9]{24})', sourcesController.del);
 
 apiRouter.get('/users', usersController.list);
+apiRouter.patch('/users/:googleID', usersController.updateRoles);
```

Comment:
we discussed with Alex in his design doc and we were in agreement to use PUT for updates instead of PATCH, patch is a bit awkward and partial updates with mongo db is the default behavior so it's trivial. Can you please change to that?

Reply:
Done.